profiles/mojoshader_profile_d3d.c

```diff
@@ -0,0 +1,676 @@
+/**
+ * MojoShader; generate shader programs from bytecode of compiled
+ *  Direct3D shaders.
+ *
+ * Please see the file LICENSE.txt in the source's root directory.
+ *
+ *  This file written by Ryan C. Gordon.
+ */
+
+#define __MOJOSHADER_INTERNAL__ 1
+#include "mojoshader_profile.h"
+
+#if SUPPORT_PROFILE_D3D
+
+const char *make_D3D_srcarg_string_in_buf(Context *ctx,
+                                                 const SourceArgInfo *arg,
```

Comment:
Going to be lots of these extra spaces since the functions are no longer static.